crates/fuel-core/src/graphql_api/service.rs

```diff
@@ -158,20 +155,14 @@ pub fn new_service(
 ) -> anyhow::Result<Service> {
     let network_addr = config.addr;
 
-    let honeycomb_enabled = config.honeycomb_enabled;
-
     let builder = schema
         .data(config)
         .data(database)
         .data(txpool)
         .data(producer)
         .data(consensus_module);
     // use honeycomb tracing wrapper if api key is configured
-    let builder = if honeycomb_enabled {
-        builder.extension(HoneyTrace)
-    } else {
-        builder.extension(Tracing)
-    };
+    let builder = builder.extension(Tracing);
```

Comment:
We can remove tracing at all because we don't use it in the testnet

Reply:
It's still useful to setup the spans used to wrap events logged within the apis